src/provider/model/AppWindow.ts

```diff
@@ -48,4 +48,6 @@ export interface AppWindow {
     focus(): Promise<void>;
 
     isReadyToReceiveIntent(intent: IntentType): Promise<boolean>;
+
+    removeAllListeners(): void;
```

Comment:
Does `test/mocks.ts` need to be updated to have this?

Reply:
Added to the mocks.